lib/app/models/module.rb

```diff
@@ -37,6 +38,13 @@ def initialize(params)
           test_sets[test_set_key] = TestSet.new(test_set_key, test_set)
         end
       end
+      if params[:sequence_requirements].present?
+        @sequence_requirements = {}.tap do |requirements|
+          params[:sequence_requirements].each do |requirement_key, texts|
```

Comment:
Why is this value called `texts`? The same thing is done in `TestingInstance`.

Reply:
changed name to `req_description`.